packages/nextjs/components/Footer.tsx

```diff
@@ -15,9 +15,9 @@ export default function Footer() {
   const configuredNetwork = getTargetNetwork();
 
   return (
-    <div className="min-h-0 p-5 flex justify-between items-center flex-col sm:flex-row gap-4">
+    <div className="min-h-0 p-5 flex justify-between items-center flex-col sm:flex-row gap-4 mb-10 lg:mb-0">
       <div>
-        <div className="flex space-x-2 fixed z-10 m-4 bottom-0 left-0">
+        <div className="flex items-center space-x-2 fixed z-20 h-16 bottom-0 left-4">
```

Comment:
We shouldn't use magic numbers (like h-16, left-4)

Height should be a result of the element size + paddings. And we shouldn't move elements with "left / right" props (only `0`)